scripts/question_answering/README.md

```diff
@@ -93,23 +93,24 @@ For reference, we've included the results from Google's Original Experiments
 |ALBERT xlarge (googleresearch/albert)  | 92.9/86.4     | 87.9/84.1    |
 |ALBERT xxlarge (googleresearch/albert) | 94.6/89.1     | 89.8/86.9    |
 
-For BERT and ELECTRA model, the results on SQuAD1.1 and SQuAD2.0 are given as follows.
+For BERT and ELECTRA model, the results on SQuAD1.1 and SQuAD2.0 are given as follows. For ELECTRA model, same hyper-parameter as original paper are used, and layer-wise learning rate decay is applied (`layerwise_decay`) with the value of 0.8 for small/base and 0.9 for large.
```

Comment:
We can paste the command of finetuning Electra with layerwise lr decay. We may just show the command for the base model.